spring-cloud-dataflow-completion/src/main/java/org/springframework/cloud/dataflow/completion/CompletionConfiguration.java

```diff
@@ -51,24 +52,22 @@
 	@Bean
 	@SuppressWarnings("unchecked")
 	public StreamCompletionProvider streamCompletionProvider() {
-		List<RecoveryStrategy<?>> recoveryStrategies = Arrays.<RecoveryStrategy<?>>asList(
-				emptyStartYieldsModulesRecoveryStrategy(),
-				expandOneDashToTwoDashesRecoveryStrategy(),
-				configurationPropertyNameAfterDashDashRecoveryStrategy(),
-				unfinishedConfigurationPropertyNameRecoveryStrategy(),
-				channelNameYieldsModulesRecoveryStrategy(),
-				modulesAfterPipeRecoveryStrategy(),
-				configurationPropertyValueHintRecoveryStrategy()
-
-		);
-		List<ExpansionStrategy> expansionStrategies = Arrays.asList(
-				addModuleOptionsExpansionStrategy(),
-				pipeIntoOtherModulesExpansionStrategy(),
-				unfinishedModuleNameExpansionStrategy(),
-				// Make sure this one runs last, as it may clear already computed proposals
-				// and return its own as the sole candidates
-				configurationPropertyValueHintExpansionStrategy()
-		);
+		List<RecoveryStrategy<?>> recoveryStrategies = new ArrayList<>();
```

Comment:
Is this change so that we can mutate the list later? I'd rather leave it as before and make a copy in the constructor.

I can change this while merging